File: leet_1.py
class Solution:
    def twoSum(self, nums: list[int], target: int) -> list[int]:
        res = []
        for i in range(len(nums)-1):
            for j in range(i+1, len(nums)):
                if nums[i] + nums[j] == target:
                    res.append(i) 
                    res.append(j) 
        return res 


nums = [3,2,4] #[2,7,11,15]

target = 6

s = Solution()
print(s.twoSum(nums=nums, target=target))

File: leet_1.ts
function twoSum(nums: number[], target: number): number[] {
    const res: number[] = []
    for (let i: number = 0; i < nums.length; i = i + 1  ){
        for (let j: number = i+1; j < nums.length; j = j + 1){
            if (nums[j] + nums[i] == target) {
                res.push(i)
                res.push(j)
            }
        }
    }
    return res 
}

const nums: number[] = [1, 2, 3, 4];
const target: number = 6;

console.log(twoSum(nums, target))